paddle/math/Matrix.cpp

```diff
@@ -3385,6 +3385,27 @@ void CpuMatrix::oneHotCrossEntropyWithSelfNormBp(Matrix& output,
   real* out = output.getData();             \
   for (size_t i = 0; i < numSamples; ++i, grad += dim, out += dim)
 
+void CpuMatrix::softmaxBackward(Matrix& outputV) {
```

Comment:
If the use `softmax` activation in this layer, this function can be removed. I suggest that we use `softmax` activation to compute in this layer.